### DISCUSSION TITLE
视频拖拽问题

### DISCUSSION THREAD

**Discussion by IAmTheReallyCoolest** [wangeditor-team/wangEditor#4802] [Q&A] [answered]
请问wangeditor5支持视频从桌面拖动到编辑器里吗，我试了图片可以直接拖动到编辑器里，视频不行， 但是wangeditor4是支持视频从桌面拖拽的(但wangeditor拖拽视频的时候，代码逻辑走的是拖拽图片的 但也能用)

**Comment by wangfupeng1988** [comment, marked as answer]
可以提交个 issue ，作为一个优化项目
